### ISSUE TITLE
Consider Different Default Log Levels 

### ISSUE THREAD

**Issue by skorfmann** [winglang/wing#2552]
### Feature Spec

- consider a different default setting for the log levels filter

### Use Cases

When first using the Wing console, it can be challenging to understand what is happening. I followed the "getting started" tutorial on the Winglang documentation (https://docs.winglang.io/getting-started/console), but I didn't receive any visual feedback when publishing a message on the queue, which made me think something was broken.

Upon investigation, I discovered that the "default levels" for the logging filter were filtering out all logs. This led me to suggest reconsidering the choice of default levels. In a development environment like this, it could be argued that users would like to see every detail. Alternatively, reevaluating the log levels for specific events could also be beneficial.

### Component

Wing Console

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

originally posted here https://github.com/winglang/wing/issues/2517#issuecomment-1549369940

**Comment by ShaiBer** [comment]
I agree that the first interactions with the application in the console should generate visible logs.
This can be achieved by either changing the default level as proposed here, or by moving some of the logs that are hidden now to a level that is visible by the current default. I think this may be a better approach in order not to overwhelm the user with too many logs at first

**Comment by skorfmann** [comment]
Just thought again that something is not working because logs were filtered by default. The fact that the log level filter setting is not persisted across restarts of the Console probably contributes to this becoming annoying.

**Comment by ShaiBer** [comment]
Yes, they should be persistent across console restarts. Probably warrants its own separate issue

**Comment by ShaiBer** [comment]
Opened #2806 to track

**Comment by staycoolcall911** [comment]
Perhaps the easiest thing would be to have the Console print a message on successful invocation of user-triggered event (e.g. "invoke function successful")

**Comment by ekeren** [comment]
@ainvoner/@skyprex/@polamoros  , what do you guys think of the above suggestion? (having the console omit messages to the to the logs)
